docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -200,8 +215,8 @@ You must be logged into Google with your `@2i2c.org` account at this point so `s
 ```
 
 ```bash
-cd ..
-sops --encrypt --in-place secrets/CLUSTER_NAME.json
+cd ../..
+sops --encrypt --in-place secrets/CLUSTER_NAME.yaml
```

Comment:
```suggestion
sops --encrypt --in-place secrets/CLUSTER_NAME.{{ json | yaml }}
```

This depends on what sort of file we're outputting. GCP's Service Accounts will be JSON, as will Azure's Service Principals when I figure that out. But raw kubeconfig files are YAML. I'm not sure of the best way to write this?

Reply:
Oooh, right! Makes sense now. Thank you!